ginga/main.py

```diff
@@ -108,6 +108,7 @@
     Bunch(module='Operations', tab='_opns', ws='operations', start=True),
     Bunch(module='WBrowser', tab='Help', ws='channels', raisekey='?', start=False),
     Bunch(module='FBrowser', tab='Open File', ws='right', start=False),
+    Bunch(module='SaveImage', tab='Save File', ws='right', start=False),
```

Comment:
This is slightly different from "Save as FITS" that you suggested but it is more consistent with "Open File".